src/common/state/redis.c

```diff
@@ -466,8 +463,8 @@ void redis_task_table_get_task(table_callback_data *callback_data) {
   int status =
       redisAsyncCommand(db->context, redis_task_table_get_task_callback,
                         (void *) callback_data->timer_id, "HGETALL task:%b",
-                        id.id, sizeof(task_id));
-  if ((status == REDIS_ERR) || db->sub_context->err) {
+                        id.id, sizeof(id.id));
+  if ((status == REDIS_ERR) || db->context->err) {
```

Comment:
This change seems a little out of the context (heh) of the rest of this commit... was it intentional?